### PR TITLE
feat: add Linea Goerli Testnet

### PR DESCRIPTION
**Pull request by ElvisKrop**
Please add Linea Goerli Testnet:

https://github.com/safe-global/safe-deployments/pull/230

Thank you.

### CODE REVIEW

File: gnosis/eth/ethereum_network.py [hunk 1]
```diff
@@ -557,6 +557,7 @@ class EthereumNetwork(Enum):
     REI_CHAIN_MAINNET = 55555
     REI_CHAIN_TESTNET = 55556
     BOBA_BNB_MAINNET = 56288
+    LINEA_GOERLI_TESTNET = 59140
```
Comment (moisses89) [inline gnosis/eth/ethereum_network.py:560]: We are using the ethereum-list name https://github.com/ethereum-lists/chains/blob/master/_data/chains/eip155-59140.json#LL2C12-L2C25 could you rename it to LINEA_TESTNET?

Comment (ElvisKrop) [reply gnosis/eth/ethereum_network.py:560]: ok, sure :wink:

Comment (ElvisKrop) [reply gnosis/eth/ethereum_network.py]: @moisses89 :heavy_check_mark:  done at [aab610f](https://github.com/safe-global/safe-eth-py/pull/560/commits/aab610fd660ca9e569dd508b25fa5797fbb7a636)

**Comment by ElvisKrop** [comment]
Happy Monday! @moisses89 
Please, check this PR once again :pray: 
Thanks in advance!

**Review by moisses89 — APPROVED** [review]

**Pull request merged**